lib/opensusebasetest.pm

```diff
@@ -800,7 +800,7 @@ sub reconnect_s390 {
         select_console('svirt');
         save_svirt_pty;
 
-        wait_serial('GNU GRUB', 180) || diag 'Could not find GRUB screen, continuing nevertheless, trying to boot';
+        wait_serial('GNU GRUB|Welcome to GRUB!', 180) || diag 'Could not find GRUB screen, continuing nevertheless, trying to boot';
```

Comment:
will this impact other jobs? Looks like touching very sensitive code broadly used.

Reply:
The same regex was used in [lib/utils.pm - reconnect_mgmt_console](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/utils.pm#L1404), and it worked fine for both DVD and Image, while I have kept the original time out settings for the Image to 0s. Should be treated as classic regex group.